mountainduck/interface.md

```diff
@@ -53,6 +53,10 @@ Bookmarks are shared between [Cyberduck](https://cyberduck.io/) and Mountain Duc
 :width: 500px
 ```
 
+### Mount point
+
+The nickname set determines the mount point used. You can customize the mount point name by editing the nickname of the bookmark.
```

Comment:
The nickname determines the display of the bookmark in the status bar menu and can be customized. Also it is used for the name of the mounted volume in Finder and drive letter on Windows respectively.